### ISSUE TITLE
(Releases) 2.0 scan speed is faster than 2.1

### ISSUE THREAD

**Issue by Goldvivi1** [AHAAAAAAA/PokemonGo-Map#2444]
2.0 scan speed is way faster than 2.1, not sure if it was intentional.

**Comment by lisandro52** [comment]
Check #2423

**Comment by mortasgus** [comment]
2.1 add a delay in each scan and the multi-threading
you can change it

In config.ini with num-threads and scan-delay

In command line
--scan-delay 1 # time delay between each requests (in seconds)
--num-threads 5 # number of threads

**Comment by Encouraging** [comment]
How can I fix this when im using Heroku?

**Comment by PaleHaunt** [comment]
Mortasgus, could you elaborate on how this done? The only config.ini file I find is config.ini.example, and I think that is not where to make the changes. Doing this with the terminal, what is the actual code to enter? Sorry, I'm a bit of a beginner... :(

**Comment by lisandro52** [comment]
@PaleHaunt 
Yes, the only config is `config.ini.example` which is the default. To use it, you should copy and rename it to `config.ini` inside the same directory, and edit the new one. The application will grab all the configuration from the new `config.ini`.

**Comment by mortasgus** [comment]
@PaleHaunt 

Like @lisandro52 say. Copy config.ini.example to config.ini and edit it.
remove the # at the begining of each line you want to be used and put a value avec the :

example :
scan-delay: 1
num-threads: 5

**Comment by naturefatcs** [comment]
So the new lines should appear as? is this correct?

num-threads:            5 # number of threads
scan-delay:             1 # time delay between each requests (in seconds)

**Comment by naturefatcs** [comment]
Afterdoing so I keep getting this error: log.error('Search thread failed. Response dictionary key error')

**Comment by mauzamora** [comment]
I get the same error but the map still works... so not really sure what it means.

**Comment by mortasgus** [comment]
@naturefatcs , yes that is correct.
A # is for a remark, a comment. So when the program read the file line by line, it will not use what so ever are after the #

"Search thread failed. Response dictionary key error" is a new issue not related to this one
